webrtc.html

```diff
@@ -6605,6 +6605,13 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       SSRC is always updated, and if the RTP packet contains CSRCs, then the
       <code><a>RTCRtpContributingSource</a></code> objects corresponding to
       those CSRCs are also updated.</p>
+      <p>All of the <code><a>RTCRtpContributingSource</a></code> objects that
+      correspond to the same <code><a>RTCRtpReceiver</a></code> MUST be
+      updated at the same time within the same microtask checkpoint, or updated
+      in a different but indistinguishable manner. This is necessary so that an
+      application can compare <code><a>RTCRtpContributingSource</a></code>
+      objects with each other, with the guarantee that they all return
+      information from the same point in time.</p>
```

Comment:
This second part is the important guarantee I think. This could perhaps be done by dictating that the algorithm MUST only be run on sets of values covering all sources, from the same point in time.